app/db/database.py

```diff
@@ -1,14 +1,15 @@
 from sqlalchemy import create_engine
-from sqlalchemy.ext.declarative import declarative_base
-from sqlalchemy.orm import sessionmaker
+from sqlalchemy.orm import sessionmaker, DeclarativeBase
 
-SQLALCHEMY_DATABASE_URL = "postgresql://postgres:postgres@localhost:5433/postgres"
+SQLALCHEMY_DATABASE_URL = "postgresql://postgres:postgres@localhost:5432/postgres"
```

Comment:
@saamo24 pls update this line. 

need to add env vars to allow read db conncetion params from envs